File: BJ.WEB/ClientApp/src/app/shared/models/history-views/get-user-games-history-view.ts
export class GetUserGamesHistoryView {

    public games: Array<GamesGetUserGamesHistoryViewItem>

    /**
     *
     */
    constructor() {
        this.games = [];

    }

}

export class GamesGetUserGamesHistoryViewItem {
    public gameId: string;
    public dateTime: Date;
    public countBots: number;
    public winner: string;
}

File: BJ.ViewModels/HistoryViews/GetUserGamesHistoryView.cs
using System;
using System.Collections.Generic;

namespace BJ.ViewModels.HistoryViews
{
    public class GetUserGamesHistoryView
    {
        public List<GameGetUserGamesHistoryViewItem> Games { get; set; }

        public GetUserGamesHistoryView()
        {
            Games = new List<GameGetUserGamesHistoryViewItem>();
        }
    }

    public class GameGetUserGamesHistoryViewItem
    {
        public Guid GameId { get; set; }
        public DateTime DateTime { get; set; }
        public int CountBots { get; set; }
        public string Winner { get; set; }
        public bool IsWinner { get; set; }

    }
}
